### PR TITLE
Support spanish tokens in the UI

### PR DESCRIPTION
**Pull request by josephcsoti**
### Description

Adds support (front-end) for the tokens:

```
yo
mi
mí
```

### CODE REVIEW

File: config/regex.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const regex = {
+  REGEX_EN_ME: /\bme\b/i,
+  REGEX_ES_MI: /\bmi\b/i,
+  REGEX_ES_MI2: /\bmí\b/i,
```
Comment (Kahtaf) [inline config/regex.js:4]: I think we are missing a version of mí. It doesn't seem to work with this mí, see Colin's Slack message: https://vana-org.slack.com/archives/C04MQM25YRK/p1675457705672829?thread_ts=1675457347.535289&cid=C04MQM25YRK

image

File: package.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
   },
   "author": "Vana (vana.com)",
   "scripts": {
-    "dev": "next dev",
+    "dev": "next dev -p 5333",
```
Comment (Kahtaf) [inline package.json:15]: Let's not include the port change

**Review by Kahtaf — CHANGES_REQUESTED** [review]
Thanks Joseph! I think we are almost there, one minor change.

File: package.json (hunk 2 above)
Comment (josephcsoti) [reply package.json:15]: whoops got sloppy with my `git add .` thnx

**Comment by josephcsoti** [comment]
Test phrases

```
pintura al oleo de mí en la playa con 3 gatos bailando
pintura realista de mi en una sala de estar de los 1970 con un perro negro
yo parado en la calle durante una tormenta con un paraguas azul
```

File: config/regex.js (hunk 1 above)
Comment (josephcsoti) [reply config/regex.js]: fixed regex (had to change it to a unicode version)

**Review by Kahtaf — APPROVED** [review]

**Pull request merged**